### ISSUE TITLE
Abstract acceptance emails seem not to arrive (and do not go to organisers)

### ISSUE THREAD

**Issue by agbuckley** [indico/indico#5362]
**Describe the bug**
For the second year I am running a conference off the CERN Indico installation, and after applying all the abstract acceptances (with the "Send notifications to submitter" box ticked in the judgement form) a majority of accepted speakers say that they did not receive any notification. This was also a problem last year.

I had to manually send an email to all abstract submitters, authors, etc. to ask them to check their abstract's web page for the decision, which then led to further administrative overhead from those who misunderstood or could not locate their abstract(s).

As organiser, it was not obvious to me either that the emails were not being sent, except that in some cases I had manually performed the submitter role, and I did not receive acceptance emails for those abstracts. I do receive other emails from this conference's Indico setup, so it seems to be an issue with these specific notifications.

As a corollary, I would add that it would be an improvement for this feature to notify all abstract authors rather than just the submitter, and to also have an option to send a copy to the conference-organiser email.

**To Reproduce**
1. Go to Management > Call for abstracts > Manage list
2. Add an abstract with yourself or a colleague as author
3. Click on the unaccepted abstract
4. Scroll down to the bottom, select "accept" (though I presume "reject" should also send a notification), assign a track and presentation type, and add a message.
5. Select "Send notifications to submitter" and click "Judge"
6. Email doesn't (?) arrive

**Expected behavior**
The email should arrive. Best that it is checked with a non-admin as author and/or submitter.

I would also expect copies to go to all authors, not just the submitter as the tickbox implies (since sometimes the submitter is the conference organiser, and the authors are the ones who really need to know), and a copy sent to the organisers (maybe a second tickbox)

**Comment by ThiefMaster** [comment]
Did you set up a notification template for abstract accepted? Unless you define one, enabling the checkbox won't do anything - it just means that if there's a matching notification template, notifications will actually be sent.

(maybe we could show a warning if no notifications have been set up)

**Comment by agbuckley** [comment]
No! I had no idea that might be necessary, so it sounds like that's definitely one to highlight in the UI! Thanks... a bit late for this time, but I'll try to bear in mind for future (and hopefully by then it'll warn me when I inevitably forget.)

On the sub-issues of mailing copies to the authors (rather than just submitter) and to the organisers -- as for e.g. registrations -- is that something that should already happen if the template is set up correctly?

**Comment by ThiefMaster** [comment]
![image](https://user-images.githubusercontent.com/179599/173065942-6e3384e3-c556-4536-b2f7-f65174a6a259.png)

Already possible when configuring the notification template :)
